app/models/folio/ability.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+class Folio::Ability
+  include CanCan::Ability
+  attr_reader :user, :site
+
+  def initialize(user, site = nil)
+    return if user.nil?
+
+    @user = user
+    @site = site
+
+    folio_console_rules
+  end
```

Comment:
tady bych pridal `rules`, ve ktere by bylo default jen `folio_console_rules`, aby to slo dobre pretezovat

Reply:
To se má dít mergováním Abiities, např
``` 
def current_ability
      @current_ability ||= Folio::Ability.new(current_user).merge(Dummy::Ability.new(current_user))
end
```

ale vzhledem k tomu, že to nakonec je i u `Folio::User#can_now?`, tak asi bude lepší přetěžování.